### PR TITLE
SEO-172160-WindowsForms-Weblink-Issue

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Have changed image alt and renamed images in md and folder.

Regards,
Yvone.

### CODE REVIEW

File: WindowsForms/chart/FAQ/How-to-add-an-image-on-chart-area.md [hunk 1]
```diff
@@ -134,4 +134,4 @@ ChartSeries chartSeries;
 
 {% endtabs %}	
 
-![Adding an image on chart area](How-to-add-image-on-chart-area/How-to-add-image-on-chart-area.png)
\ No newline at end of file
+![Adding an image on chart area in WindowsForms application](how-to-add-image-on-chart-area/windowsforms-chart-adding-image-on-chart-area.png)
```
Comment (MallikaRK) [inline WindowsForms/chart/FAQ/How-to-add-an-image-on-chart-area.md:137]: Don't use repeated keywords in image name. you used chart keyword in 2 places.
adding-image-on-winforms-chart-area - is fine.
update it.

File: WindowsForms/chart/FAQ/How-to-display-custom-tooltip-over-Histogram-Chart.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ End Sub
 
 {% endtabs %}	
 
-![Chart Histogram](How-to-display-custom-tooltip-over-Histogram-Chart_images/How-to-display-custom-tooltip-over-Histogram-Chart_img1.jpeg)
+![Chart Histogram in WindowsForms application](how-to-display-custom-tooltip-over-histogram-chart_images/windowsforms-charthistogram-chart.jpeg)
```
Comment (MallikaRK) [inline WindowsForms/chart/FAQ/How-to-display-custom-tooltip-over-Histogram-Chart.md:66]: Image name seems wrong. check and update.
windowsforms-histogram-chart

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/chart/FAQ/How-to-add-an-image-on-chart-area.md (hunk 1 above)
Comment (Yvone-Atieno) [reply WindowsForms/chart/FAQ/How-to-add-an-image-on-chart-area.md:137]: changed

File: WindowsForms/chart/FAQ/How-to-display-custom-tooltip-over-Histogram-Chart.md (hunk 2 above)
Comment (Yvone-Atieno) [reply WindowsForms/chart/FAQ/How-to-display-custom-tooltip-over-Histogram-Chart.md]: changed

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/chart/FAQ/How-to-add-an-image-on-chart-area.md [hunk 3]
```diff
@@ -134,4 +134,4 @@ ChartSeries chartSeries;
 
 {% endtabs %}	
 
-![Adding an image on chart area](How-to-add-image-on-chart-area/How-to-add-image-on-chart-area.png)
\ No newline at end of file
+![Adding an image on chart area in WindowsForms application](how-to-add-image-on-chart-area/windowsforms-adding-image-on-chart-area.png)
```
[the remark this answers is not in the document]
Comment (Yvone-Atieno) [reply WindowsForms/chart/FAQ/How-to-add-an-image-on-chart-area.md:137]: changed

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**